pyaerocom/data/data_sources.ini

```
@@ -52,3 +52,9 @@ dataset_name = GHOST
 ts_type_src=daily
 has_overlaps=False
 stat_merge_pref_attr = ts_type
+
+[GHOST.monthly]
+dataset_name = GHOST
+ts_type_src=daily
```

Shouldn't this be:
```suggestion
ts_type_src='monthly'
```
?